src/planwise/model/projects2.clj

```diff
@@ -30,7 +32,14 @@
 (s/def ::capacity number?)
 (s/def ::providers (s/keys :req-un [::capacity]))
 
-(s/def ::config (s/nilable (s/keys :req-un [::demographics ::actions ::coverage ::providers])))
+(defmulti attr-actions :analysis-type)
+(defmethod attr-actions "budget" [_]
+  (s/keys :req-un [::analysis-type ::actions]))
+(defmethod attr-actions "action" [_]
+  (s/keys :req-un [::analysis-type]))
+
+(s/def ::config-base (s/keys :req-un [::demographics ::coverage ::providers]))
```

Comment:
shouldn't analysis-type validation be part of config-base?

Reply:
I'll add it. It should fail at attr-actions but it makes more sense to fail early in config-base.